Dockerfile

```diff
@@ -1,22 +1,19 @@
-# ubuntu 16.04 xenial image
-FROM ubuntu
+# use base python
+FROM python:2.7
 
-# apt update it
-RUN apt-get update
-# get essentials
-RUN apt-get install -y tar git curl wget dialog net-tools build-essential python python-dev python-distribute python-pip
```

Comment:
wow. I really thought that image was kind of minimal. @purukaushik can you confirm the app doesn't need those extra packages?